zeppelin-web/pom.xml

```diff
@@ -120,34 +152,77 @@
           </execution>
 
           <execution>
-            <id>yarn install</id>
-            <goals>
-              <goal>yarn</goal>
-            </goals>
-            <configuration>
-              <arguments>install --no-lockfile</arguments>
-            </configuration>
+              <id>yarn add bower</id>
+              <goals>
+                  <goal>yarn</goal>
+              </goals>
+              <configuration>
+                <!--ignore-scripts-->
+                  <arguments>add bower --no-lockfile --no-bin-links</arguments>
```

Comment:
@Leemoonsoo which box do you use? Could you give me list of preinstalled packages?
I have used empty headless vagrant images for Centos7 and Ubuntu16 and I have had problem because there were no required packages. 
More over at http://zeppelin.apache.org/docs/snapshot/install/build.html#install-requirements
we have list of these required packages.